submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+- **How long you will keep using the burner devices:** If you are going to switch to electrical machines and alternate power sources soon, it may not be worth the effort of setting up a large fuel processing system.
+
+Since the above factors depend on your map and your plans, the conclusion varies on whether fuel processing is useful. Based on my experiences, I can offer the following advice about processing fuels:
+
+- **Processed fuels are excellent for trains and tanks.** The acceleration gains from rocket fuel is quite satisfying (+80%) and nuclear fuel is on another level... (+150%)
+- **You already need to mass produce solid fuel and rocket fuel to make rocket parts,** so it is not too difficult to produce a little more for your vehicle or other burner device needs.
+- **If you are burning coal for power, processing it into solid fuel will cut the coal demand by half.** And of course, you can go all out and run your boilers with nuclear fuel to reduce the demand to about 1/20th. However, unlocking coal liquification or nuclear fuel requires production science, which means you can unlock solar and (regular) nuclear power much sooner and [their advantages](https://www.reddit.com/r/technicalfactorio/comments/srosza/coal_vs_solar_vs_nuclear_setup_costs_and_running/) may make it worthwhile to switch to them before upgrading your coal setup. Nuclear power is [easier to do than it may seem]([Alt-F4 #57 - Nuclear Energy and You](https://alt-f4.blog/ALTF4-57/#)), as it provides massive energy yields for very few reactors and centrifuges, without needing Kovarex Enrichment!
+
+## Summary
+
+This article started with a little burning question that brought more burning questions and evolved into an exploration of all of Factorio's fuels. We uncovered the following:
+
+**(1) Coal vs. coal liquification:** Liquifying coal and directly converting it into solid fuel yields 57% extra energy per coal, when no modules are used.
+
+**(2) Coal liquification with cracking:** If we also crack the heavy oil before making the solid fuel, it yields 103% extra energy per coal, when no modules are used.
+
+**(3) Coal vs. oil for solid fuel production:** Coal produces about 8 times as much solid fuel or energy as crude oil per unit, when no modules are used. However, you can transport about 60% more energy in a fluid wagon of oil than a cargo wagon of coal. Meanwhile, coal has many exclusive uses it may be needed for instead of making oil products, while crude oil has only one such exclusive use. Therefore using oil for your solid fuel production may be more practical, but it ultimately depends on the abundances of coal and oil on your map.
```

Comment:
```suggestion
**(3) Coal vs. oil for solid fuel production:** Coal produces about eight times as much solid fuel or energy as crude oil per unit, when no modules are used. However, you can transport about 60% more energy in a fluid wagon of oil than a cargo wagon of coal. Meanwhile, coal has many exclusive uses it may be needed for instead of making oil products, while crude oil has only one such exclusive use. Therefore using oil for your solid fuel production may be more practical, but it ultimately depends on the abundances of coal and oil on your map.
```